python-sdk/nuscenes/nuscenes.py

```diff
@@ -1917,12 +1944,25 @@ def render_scene_with_pointclouds_for_all_cameras(self, scene_token: str, out_pa
                 # have points and others will not.
                 points, coloring, im = self.map_pointcloud_to_image(pointsensor_token, camera_token,
                                                                     render_intensity=False,
-                                                                    show_lidarseg_labels=True,
+                                                                    show_lidarseg=True,
                                                                     filter_lidarseg_labels=filter_lidarseg_labels,
                                                                     lidarseg_preds_bin_path=lidarseg_preds_bin_path)
 
                 if im is not None:
-                    mat = plt_to_cv2(points, coloring, im, imsize)
+                    if with_anns:
+                        # Get annotations and params from DB.
+                        impath, boxes, camera_intrinsic = self.nusc.get_sample_data(camera_token,
+                                                                                    box_vis_level=BoxVisibility.ANY)
+                        h, w, c = cv2.imread(impath).shape
+
+                        mat = plt_to_cv2(points, coloring, im, (w, h), dpi=200)
```

Comment:
Let's use the same standard dpi setting everywhere (could make it a parameter).

Reply:
(w, h) == imsize, or?

Reply:
`dpi=200` is a parameter which is manually found to make the lidar dots most visually appealing when plotted with the boxes. I would suggest not to allow users to change it as this may mess up the nice-looking-ness of our renderings

`(w,h) != imsize`; `w` and `h` are the image's original height and width as the boxes returned by `get_sample_data` are scaled wrt to that. This also explains why the dpi value is different here compared to everywhere else, as it has to be scaled wrt to the results returned by `get_sample_data`.

I've added comments in the script to explain this as well.